### ISSUE TITLE
Issues in Backend for Login Page

### ISSUE THREAD

**Issue by sghoshjr** [vicennial/Group-G#9]
By default, the webpage redirects to login page but it does not store the session. (cookie is not set)
i.e /machine.html is still accessible

(Also, now machine.html is the manual interface and index.html is the login page)

There aren't issues when Correct uid and pwd are entered. 
1. But wrong pwd/uid closes the nodejs server.
2. Machine.html is accessible and requires no authentication.

PS : Please create a db called "isw" in mongodb with a test user { uid : "Machine1", pwd: "test123"} before testing.

**Comment by sghoshjr** [comment]
Update [#65](https://github.com/vicennial/Group-G/commit/62ecea717f02b2495f42218b9f915cabde5a9ac5) : Wrong Passwords do not terminate NodeJS Server

**Comment by sghoshjr** [comment]
Update c96d12f5fe52044d2b5d600da917a633128457a6 
Cookies are added to enable sessions.
MainPage can be accessed directly if session is in progress. Else Redirects to Login Page.

You can logout via localhost:3500/Logout. It clears the session and redirects to login page.

**Comment by sghoshjr** [comment]
Still /index.html is accessible and asks for login-id again even if cookie is set.

**Comment by vicennial** [comment]
Issue:
Redirect does not work when using a different user from the one mentioned in the topic head(Machine1).

**Comment by vicennial** [comment]
Commit 5a0191e4581370a4c60230f9a1e0fade904dd042 temporarily avoid this issue. 
(User is redirected to machine.html instead of MainPage)

**Comment by sghoshjr** [comment]
```js
app.get('/machine.html',function(req,res)
	{
		console.log("Redirect");
		res.redirect('index.html');
	});
```
Also this thing doesn't work. I was hoping to have it check for cookie Id before giving access to machine.html or else redirect. But this thing is never triggered.  

i.e. explicitly typing localhost/machine.html always displays the page.

**Comment by sghoshjr** [comment]
Closed Issue

**Comment by sghoshjr** [comment]
Fixed commit 0106b1c
